### DISCUSSION TITLE
My GitHub account has been blocked.

### DISCUSSION THREAD

**Discussion by vitohuangdev** [vercel/community#2650] [Help] [answered]
### Summary

My GitHub account has been blocked as the screenshot shown below.

![image](https://github.com/vercel/community/assets/108714402/363ab8e2-8a70-4563-8f3c-aa9b9ebc1fa2)

### Steps to Reproduce

Please unblock my account access. (I've contacted registration@vercel.com)

**Comment by swarnava** [comment, marked as answer]
Hi, you need to contact to the email address mention in the error message. :)
